CONTRIBUTING.rst

```diff
@@ -179,8 +143,8 @@ Before you submit a pull request, check that it meets these guidelines:
 
 1. The pull request should include tests for functionality that is not already
    tested. We strive for 100% test coverage and pull requests should not add any
-   new untested code. You can generate coverage reports locally by running the
-   ``tox`` tests.
+   new untested code. You can generate coverage reports locally by running
+   ``pytest`` through ``coverage`` as ``uv run --group covcheck coverage run -m pytest``
```

Comment:
Tox used to create the html I think, the command as you have it will only create the `.coverage` statistics file, would be good to mention how to get human readable coverage stats: 

```suggestion
   new untested code. You can generate coverage reports locally by first generating coverage statistics with
   ``pytest`` through ``coverage`` as ``uv run --group covcheck coverage run -m pytest``. To generate reports from the coverage statistics, you can use ``uv run coverage report`` to print to screen or ``uv run coverage html`` to generate an html that you can open in a browser. 
```